manifests/django.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
   name: django
   namespace: bennuhp
 spec:
-  replicas: 1
+  replicas: 3
```

Comment:
note: same as MySQL, which have been implemented as StatefulSet and scheduled onto each node, for ensuring access to Django will be enclosed within a node for accessing MySQL Pod, we have scaled out as 3.